shared/actions/json/gregor.json

```diff
@@ -0,0 +1,9 @@
+{
+    "prelude": "import * as Constants from '../constants/gregor'\nimport * as RPCTypes from '../constants/types/flow-types'\nimport * as GregorTypes from '../constants/types/flow-types-gregor'",
```

Comment:
:sheep: could also make this an array of imports to improve readability (see https://stackoverflow.com/a/21333578, it's frowned on in the comments but might make sense for this use since it'd be a meaningful split)

Reply:
i actually already did this in the next prs ;)